main/remoteservices/src/EBox/RemoteServices/RESTClient.pm

```diff
@@ -259,7 +260,19 @@ sub request {
     $ua->agent("ZentyalServer $version");
     $ua->ssl_opts('verify_hostname' => EBox::Config::boolean('rs_verify_servers'));
     # Set HTTP proxy if it is globally set as environment variable
-    $ua->proxy('https', $ENV{HTTP_PROXY}) if (exists $ENV{HTTP_PROXY});
+
+    my $proxy =  $ENV{HTTP_PROXY};
+    if (not $proxy) {
+        # check that if we have proxy set in network RO
+        my $network = EBox::Global->getInstance(1)->modInstance('network');
+        $proxy = $network->proxySettings(1);
```

Comment:
I do not like to use strings as bool values because the developer reading the code may think that it is a option name and the behavior of the method could be changed with other string.
Either way he will need to read the parameters line or the documentation, so I stick with a simple bool value,